SQRLPlatformAwareInstaller/Platform/OSX/Installer.cs

```diff
@@ -81,5 +87,40 @@ public DownloadInfo GetDownloadInfoForAsset(GithubRelease selectedRelease)
                 DownloadUrl = selectedRelease.assets.Where(x => x.name.Contains("osx-x64.zip")).First().browser_download_url
             };
         }
+
+        /// <summary>
+        /// Moves the Db from the current location to the new user space location
+        /// </summary>
+        /// <param name="currentPath"></param>
+        /// <returns></returns>
+        public bool MoveDb(string currentPath)
```

Comment:
Are we really going to duplicate this code for each platform?

Reply:
It is slightly different for windows, I wasn't sure how to work around that... I guess I could do a Platform check in there... I'll do that!

Reply:
Ah, haven't checked that closely, sorry. Maybe you'll find a good way still.

Reply:
Yeah this is now fixed.